kafka/admin/config_resource.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+
+from enum import IntEnum
```

Comment:
We have a pretty strict rule against mandatory external dependencies. I know this is standard in py3, but it isn't in py2. Can we avoid using it? If not, we should vendor the py27 version (see kafka/vendor)

Reply:
ugh... people are still using python 2?  Isn't it EOL?  Hrmph...  lemme look at it.

Reply:
Any progress @llamahunter ?

And yes, I know of multiple companies who heavily rely on this library and still have large codebases in python 2. Most are planning to migrate to python 3, but still in the early stages. I'd guess it'll be another 2-3 years before I'd be okay with saying that we as a core infra library are going to stop supporting python 2 in our releases. I look forward to that day, it's just not quite here yet.

Reply:
FWIW, I'd love to see this be part of the project as well.